include/boost/url/params.hpp

```diff
@@ -71,7 +71,7 @@ class params
     /** A random-access iterator referencing parameters in a url query.
 
         This iterator lazily constructs instances of @ref params::reference, which
-        contain @ref string_value decoded representations of the current <key, value> pair
+        contain @ref const_string decoded representations of the current <key, value> pair
```

Comment:
You have to be careful putting `<` and `>` in javadocs. It is usually better not to do that, because it will confuse the toolchain.